kotsadm/pkg/render/render.go

```diff
@@ -132,6 +141,7 @@ func RenderDir(archiveDir string, a *apptypes.App, downstreams []downstreamtypes
 		IsAirgap:         a.IsAirgap,
 		AppSlug:          a.Slug,
 		IsGitOps:         a.IsGitOps,
+		AppSequence:      a.CurrentSequence + 1,
```

Comment:
why +1? maybe justify this in a comment

Reply:
+1 because it provides the most recent sequence, and we want the next sequence here